### ISSUE TITLE
VST3 validation will not run if it is called with the full directory path to the vst3 (with trailing slash

### ISSUE THREAD

**Issue by cjappl** [Tracktion/pluginval#20]
Steps to repro

1. Have a VST3 plugin (this example, using FabFilter Pro-Q 2)
2. Run command `/Applications/pluginval.app/Contents/MacOS/pluginval --validate /Library/Audio/Plug-Ins/VST3/FabFilter\ Pro-Q\ 2.vst3/`

Expected result:
Plugin validation runs as expected

Observed result:
failure, no plugin found. Output error:
```
Started validating: /Library/Audio/Plug-Ins/VST3/FabFilter Pro-Q 2.vst3/
Random seed: 0x750eab4
Validation started: 25 Jul 2019 1:04:48pm

Strictness level: 5
-----------------------------------------------------------------
Starting test: pluginval / Scan for known types: /Library/Audio/Plug-Ins/VST3/FabFilter Pro-Q 2.vst3/...
Num types found: 0
!!! Test 1 failed: No types found
FAILED!!  1 test failed, out of a total of 1

Finished validating: /Library/Audio/Plug-Ins/VST3/FabFilter Pro-Q 2.vst3/
*** FAILED: 1 TESTS
*** FAILED: 1 TESTS

```

The trailing slash is the culprit here. Runnign the same command without the trailing slash (`/Applications/pluginval.app/Contents/MacOS/pluginval --validate /Library/Audio/Plug-Ins/VST3/FabFilter\ Pro-Q\ 2.vst3`) will run as intended.

A possible fix could be just stripping any slashes off the end of inputted files.

**Comment by drowaudio** [comment]
Ok, this should be fixed on develop now here: https://github.com/Tracktion/pluginval/commit/28a4a9db1b39067337494194b63fcf7811aff20c

I've put it slightly lower down the chain to catch it in more places but tbh I think this is more an issue with the shell you're using adding trailing "/". That really changes the semantics of some apps.

Let me know if there any any issues with my fix.

**Comment by cjappl** [comment]
Looks great!! Testing now

**Comment by cjappl** [comment]
Tested and confirmed working. Thanks so much!

Yeah, agreed. I didn't actually realize it was doing that until I ran into this issue. I'll have to keep an eye out in the future if anything else breaks with this. I think it has to do with the way mac does applications/plugins as directories under the hood.

I tested in my normal shell (fish) as well as bash, and both tab complete with the slash.

Thanks again for a quick fix, extremely helpful!

Have a good one.
